test/core/util/tls_utils.h

```diff
@@ -130,7 +131,8 @@ class AsyncExternalVerifier {
                     void* callback_arg, grpc_status_code* sync_status,
                     char** sync_error_details);
 
-  static void Cancel(void*, grpc_tls_custom_verification_check_request*) {}
+  static void Cancel(void*, grpc_tls_custom_verification_check_request*,
```

Comment:
We just don't care about defining more than a no-op cancels because it's a test helper and we aren't testing cancel in this area?

Reply:
We explicitly call `Cancel` in the test code, unlike in some cases we implement the interface and expect it to be called directly and correctly. So we don't need to do anything here.